### PR TITLE
cdc: add some explanations about lossy DDL

### PR DESCRIPTION
**Pull request by hi-rustin**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-faq.md [hunk 1]
```diff
@@ -202,6 +202,17 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+## TiCDC 是否会将有损 DDL 造成的数据变更同步到下游？
+
+有损 DDL 是指在 TiDB 中执行可能会造成数据发生改变的 DDL，一些常见的有损 DDL 操作如下：
+
+- 修改列的类型，例如：INT -> VARCHAR
+- 修改列的长度，例如：VARCHAR(20) -> VARCHAR(10)
+- 修改列的精度，例如：DECIMAL(10, 3) -> DECIMAL(10, 2)
+- 修改列的符号（有符号数/无符号数），例如：INT UNSIGNED -> INT SIGNED
+
+在 TiDB v7.1.0 之前，TiCDC 会将一条新旧数据相同的 DML 事件同步到下游，对于下游是 MySQL 的情况，这些 DML 事件不会造成任何数据变更。直到下游接收到该 DDL 语句并执行后，才会造成数据变更。但是对于下游是 Kafka 或者云存储的情况，会导致 TiCDC 写入一条无用的数据到下游。在 TiDB v7.1.0 之后，TiCDC 会过滤掉这些无用的 DML 事件，不会将其同步到下游。
```
Comment (Oreoxmt) [inline ticdc/ticdc-faq.md:214]:
```suggestion
## TiCDC 是否会将有损 DDL 造成的数据变更同步到下游？

有损 DDL 是指在 TiDB 中执行可能会导致数据改变的 DDL。一些常见的有损 DDL 操作包括：

- 修改列的类型，例如：INT -> VARCHAR
- 修改列的长度，例如：VARCHAR(20) -> VARCHAR(10)
- 修改列的精度，例如：DECIMAL(10, 3) -> DECIMAL(10, 2)
- 修改列的符号（有符号数/无符号数），例如：INT UNSIGNED -> INT SIGNED

在 TiDB v7.1.0 之前，TiCDC 会将一条新旧数据相同的 DML 事件同步到下游。当下游是 MySQL 时，这些 DML 事件不会造成任何数据变更，只有下游接收并执行该 DDL 语句后，数据才会发生变更。但是当下游是 Kafka 或者云存储时，这会导致 TiCDC 写入一条无用的数据到下游。

从 TiDB v7.1.0 开始，TiCDC 会过滤掉这些无用的 DML 事件，不再将它们同步到下游。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-faq.md [hunk 2]
```diff
@@ -202,6 +202,19 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
 4. 建立一个新的 changefeed，从 `BackupTS` 开始同步任务。
 5. 删除旧的 changefeed。
 
+## TiCDC 是否会将有损 DDL 造成的数据变更同步到下游？
```
Comment (hfxsd) [inline ticdc/ticdc-faq.md:205]:
```suggestion
## TiCDC 是否会将有损 DDL 产生的数据变更同步到下游？
```

File: ticdc/ticdc-faq.md [hunk 3]
```diff
@@ -202,6 +202,19 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+有损 DDL 是指在 TiDB 中执行可能会导致数据改变的 DDL。一些常见的有损 DDL 操作包括：
+
+- 修改列的类型，例如：INT -> VARCHAR
+- 修改列的长度，例如：VARCHAR(20) -> VARCHAR(10)
+- 修改列的精度，例如：DECIMAL(10, 3) -> DECIMAL(10, 2)
+- 修改列的符号（有符号数/无符号数），例如：INT UNSIGNED -> INT SIGNED
+
+在 TiDB v7.1.0 之前，TiCDC 会将一条新旧数据相同的 DML 事件同步到下游。当下游是 MySQL 时，这些 DML 事件不会造成任何数据变更，只有下游接收并执行该 DDL 语句后，数据才会发生变更。但是当下游是 Kafka 或者云存储时，这会导致 TiCDC 写入一条无用的数据到下游。
```
Comment (hfxsd) [inline ticdc/ticdc-faq.md:214]:
```suggestion
在 TiDB v7.1.0 之前，TiCDC 会将一条新旧数据相同的 DML 事件同步到下游。当下游是 MySQL 时，这些 DML 事件不会产生任何数据变更，只有下游接收并执行该 DDL 语句后，数据才会发生变更。但是当下游是 Kafka 或者云存储时， TiCDC 会写入一条无用的数据到下游。
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-faq.md [hunk 4]
```diff
@@ -202,6 +202,19 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+## TiCDC 是否会将有损 DDL 产生的数据变更同步到下游？
+
+有损 DDL 是指在 TiDB 中执行可能会导致数据改变的 DDL。一些常见的有损 DDL 操作包括：
+
+- 修改列的类型，例如：INT -> VARCHAR
+- 修改列的长度，例如：VARCHAR(20) -> VARCHAR(10)
+- 修改列的精度，例如：DECIMAL(10, 3) -> DECIMAL(10, 2)
+- 修改列的符号（有符号数/无符号数），例如：INT UNSIGNED -> INT SIGNED
+
+在 TiDB v7.1.0 之前，TiCDC 会将一条新旧数据相同的 DML 事件同步到下游。当下游是 MySQL 时，这些 DML 事件不会产生任何数据变更，只有下游接收并执行该 DDL 语句后，数据才会发生变更。但是当下游是 Kafka 或者云存储时， TiCDC 会写入一条无用的数据到下游。
```
Comment (Oreoxmt) [inline ticdc/ticdc-faq.md:214]:
```suggestion
在 TiDB v7.1.0 之前，TiCDC 会将一条新旧数据相同的 DML 事件同步到下游。当下游是 MySQL 时，这些 DML 事件不会产生任何数据变更，只有下游接收并执行该 DDL 语句后，数据才会发生变更。但是当下游是 Kafka 或者云存储时，TiCDC 会写入一条无用的数据到下游。
```

File: releases/release-7.1.0.md [hunk 5]
```diff
@@ -262,6 +262,9 @@ TiDB 版本：7.1.0 (upcoming)
     - 提升 TiFlash 在存算分离架构下的性能和稳定性 [#6882](https://github.com/pingcap/tiflash/issues/6882) @[JaySon-Huang](https://github.com/JaySon-Huang) @[breezewish](https://github.com/breezewish) @[JinheLin](https://github.com/JinheLin)
     - 支持在 Semi Join 或 Anti Semi Join 中，通过选择较小的表作为 Build 端来优化查询性能 [#7280](https://github.com/pingcap/tiflash/issues/7280) @[yibin87](https://github.com/yibin87)
 
++ TiCDC
+    - TiCDC 过滤了由有损 DDL 语句引起的数据变更，避免发送无效数据变更 [#43227](https://github.com/pingcap/tidb/issues/43227) @[hi-rustin](https://github.com/hi-rustin)
```
Comment (Oreoxmt) [inline releases/release-7.1.0.md]:
```suggestion

    - TiCDC 过滤了由有损 DDL 语句引起的数据变更，避免发送无效数据变更 [#43227](https://github.com/pingcap/tidb/issues/43227) @[hi-rustin](https://github.com/hi-rustin)
```

File: releases/release-7.1.0.md [hunk 6]
```diff
@@ -262,6 +262,9 @@ TiDB 版本：7.1.0 (upcoming)
     - 提升 TiFlash 在存算分离架构下的性能和稳定性 [#6882](https://github.com/pingcap/tiflash/issues/6882) @[JaySon-Huang](https://github.com/JaySon-Huang) @[breezewish](https://github.com/breezewish) @[JinheLin](https://github.com/JinheLin)
     - 支持在 Semi Join 或 Anti Semi Join 中，通过选择较小的表作为 Build 端来优化查询性能 [#7280](https://github.com/pingcap/tiflash/issues/7280) @[yibin87](https://github.com/yibin87)
 
++ TiCDC
```
Comment (Oreoxmt) [inline releases/release-7.1.0.md:265]:
```suggestion
+ TiCDC

```

**Review by Oreoxmt — COMMENTED** [review]
LGTM

File: releases/release-7.1.0.md [hunk 7]
```diff
@@ -262,6 +262,10 @@ TiDB 版本：7.1.0 (upcoming)
...
+
+    - TiCDC 过滤了由有损 DDL 语句导致的数据变更，避免发送无效数据变更 [#43227](https://github.com/pingcap/tidb/issues/43227) @[hi-rustin](https://github.com/hi-rustin)
```
Comment (Oreoxmt) [inline releases/release-7.1.0.md:267]: @hfxsd PTAL

Comment (hfxsd) [reply releases/release-7.1.0.md:267]: LGTM

**Pull request merged**